app/models/page/renderer.rb

```diff
@@ -22,6 +22,8 @@ def render(text, options = {})
     doc = add_code_filenames(doc)
     doc = add_callout(doc)
     doc = init_responsive_tables(doc)
+    doc = add_copy_to_clipboard_button(doc)
```

Comment:
Here's an idea: what are your thoughts around adding the copy button with JavaScript rather than using the renderer?

I think this is worth a try because:

1. Separation of concern: renderer's focus is to render the actual content, whilst JavaScript provides UI functionality
2. Less work for the renderer, let the client-side do the work

Reply:
Good call out. I think we probably still want to attach some kind of data attribute to make the javascript not dependant on CSS classes since this often leads to unintentional breakages but I think it's a good suggestion based on where it landed 👍🏻